### PR TITLE
Remove inaccurate "is already enabled" message

### LINKED ISSUE #22389: Enabling an app via CLI, 'appname enabled' message should not be shown if the app was already enabled because it is redundant
### Steps to reproduce

1. Try to enable an app that was already enable via CLI (e.g. sudo -u www-data ./occ app:enable files_external)

### Expected behaviour

A message 'files_external is already enabled' should be shown

### Actual behaviour
It is shown two messages:
- files_external is already enabled
- files_external enabled

### Server configuration
**Operating system**: ubuntu 14.04

**Web server:** apache 2.4.7

**Database:** mysql

**PHP version:** 5.5.9

**ownCloud version: ownCloud Enterprise Edition 9.0 beta 1 (daily) Build:2016-02-15T04:14:27+00:00 ac367b97f0567481d3496a213c04eeb5cf654a9f

**Updated from an older ownCloud or fresh install:** Fresh install

**Are you using external storage, if yes which one:** no;
**Are you using encryption:** no
**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** 

oC 9.0

screen shot 2016-02-15 at 11 55 48

oC 8.2.2 (stable)

screen shot 2016-02-15 at 11 56 34

### PR DESCRIPTION
**Pull request by nickvergessen**
@davitol @MorrisJobke @LukasReschke 

Fix #22389

### CODE REVIEW

File: core/command/app/enable.php [hunk 1]
```diff
@@ -70,10 +70,6 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 		}
 
 		$groups = $input->getOption('groups');
-		if ($this->manager->isInstalled($appId) && empty($groups)) {
-			$output->writeln($appId . ' is already enabled');
```
Comment (nickvergessen) [inline core/command/app/enable.php]: This message should only be shown, if the old and the new setting are the same.
The problem is the API has no way to check this.

**Comment by davitol** [comment]
Tested and as @nickvergessen said in https://github.com/owncloud/core/issues/22389#issuecomment-184604819 now it is only shown 'app enabled' message and already enabled message is not longer shown.

**Comment by nickvergessen** [comment]
Yeah, the problem is we can't find out if the app was enabled for a group(s) or all. Only if it was enabled for anything or not. For disable that is enough, but for the enable case it doesn't work

**Comment by MorrisJobke** [comment]
Tested and works :+1:

**Pull request merged**